### PR TITLE
Add lat long for provider to the public api

### PR DESCRIPTION
**Pull request by davidgisbey**
### Context

We're now using the public API so I've close the other PR, added lat/long here and updated the docs accordingly.

### Changes proposed in this pull request

- Add lat & long to the provider serialiser
- Update tests
 
### Guidance to review

Are there some docs that need updating somewhere @dankmitchell?

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Review by gpeng — CHANGES_REQUESTED** [review]
Thanks for adding this @davidgisbey. The docs need updating here -> https://github.com/DFE-Digital/teacher-training-api/blob/master/swagger/public_v1/component_schemas/ProviderAttributes.yml

**Comment by davidgisbey** [comment]
Thanks for pointing me in the right direction Graham

**Review by TobyRet — APPROVED** [review]

**Review by defong — APPROVED** [review]
You may need to update swagger/public_v1/api_spec.json as well

File: swagger/public_v1/component_schemas/ProviderAttributes.yml [hunk 1]
```diff
@@ -41,6 +41,14 @@ properties:
     description: "The provider's postcode"
     maxLength: 8
     example: SK2 6AA
+  latitude:
+    type: float
+    description: "The latitude of the provider's address"
+    example: "51.498161"
+  longitude:
+    type: float
+    description: "The latitude of the provider's address"
```
Comment (gpeng) [inline swagger/public_v1/component_schemas/ProviderAttributes.yml:50]:
```suggestion
    description: "The longitude of the provider's address"
```

**Review by gpeng — CHANGES_REQUESTED** [review]

**Pull request merged**